broccoli/demo.js

```diff
@@ -2,16 +2,25 @@
 'use strict';
 
 var funnel = require('broccoli-funnel');
+var mergeTrees = require('broccoli-merge-trees');
 
 module.exports = function() {
   var demoDir = 'demo';
-  return funnel(demoDir, {
-    include: [
-      '**/*.css',
-      '**/*.js',
-      'favicon.ico',
-      '**/*.html'
-    ],
-    destDir: '/demo'
-  });
+  return mergeTrees([
+    funnel(demoDir, {
+      include: [
+        '**/*.css',
+        '**/*.js',
+        'favicon.ico',
+        '**/*.html'
+      ],
+      destDir: '/demo'
+    }),
+    funnel('./node_modules/mobiledoc-dom-renderer/dist/global/', {
```

Comment:
I believe it's safest to use path and `require.resolve` here — in npm 3, for instance, deduping may mean we can't guarantee the filepath to the node module for this.

```
// this will be the directory to the file that the renderer's package.json `main` points at
var rendererDir = require('path').dirname( require.resolve('mobiledoc-dom-renderer') );
funnel(rendererDir, {
  include: ['relative/path/from/main/to/mobile-doc-renderer-global'],
  ...
});
```